### ISSUE TITLE
No Bounty Payout - Paid to the wrong wallet address

### ISSUE THREAD

**Issue by AVeryTastyRaspberry** [mysteriumnetwork/node#4244]
My node shows that it should have had a Bounty Payout, however, I have not received the payout.   When I looked at the Bounty Payout info and clicked on the etherscan.io to look at the transaction, it should it was paid out to a different wallet address than the one stored on the node.

Node ID:   0xb6ff91c5a4e2a40d329e382e76d1fb8409e271b9

**Comment by AVeryTastyRaspberry** [comment]
I forgot to say that the wallet address it was paid to does not belong to me.

**Comment by tomasmik** [comment]
Hi, sorry that this has happened. Are you sure that either you or someone else didn't change your wallet? What is the wallet displayed in node-ui or the CLI?

**Comment by tomasmik** [comment]
You were paid to this wallet: 0xc4f3Ef3d75BCeFEBe60e91D81709F54e72C32cb6 and youre wallet has not been changed for months now according to the database.

**Comment by AVeryTastyRaspberry** [comment]
The only time the wallet address would have been changed is when I was migrating from RPI to Docker but that should have been from blank to 0xc4f3Ef3d75BCeFEBe60e91D81709F54e72C32cb6.   Looks like we both have the same details for the wallet address,  but that is not what is displayed on the bounty payout history screen and I can confirm I did not receive any sum to the confirmed wallet address ( 0xc4f3Ef3d75BCeFEBe60e91D81709F54e72C32cb6 ) the last transaction received for that wallet was ~9th February 2021, which would be easy to check on your end.

[Etherscan.io - transaction - displayed in my bounty payout history](https://etherscan.io/tx/0x2a738b83939971e2f7a5b3b1475ad4c849909a803329334cfa31da773310ad75).   You will see that transaction does not relate to the wallet address you and I have.
